vex-riscv/src/Utils/Print.hs

```diff
@@ -0,0 +1,46 @@
+-- SPDX-FileCopyrightText: 2022 Google LLC
+--
+-- SPDX-License-Identifier: Apache-2.0
+
+-- | Println-debugging during simulation
+module Utils.Print (hookPrint, getDataBytes) where
+
+import Clash.Prelude
+import qualified Data.ByteString as BS
+import Data.Foldable (traverse_)
+import qualified Data.List as L
+import Data.Maybe (catMaybes)
+import Data.Word (Word8)
+
+getDataBytes ::
+  -- | How many bytes to sample
```

Comment:
Yes, those functions already exist in `contranomy-sim`, but because I didn't want to create a dependency from `vex-riscv` on `contranomy-sim` I copied this module (and also the ELF loading one). If we eventually want to fully replace contranomy then I think it makes sense that those are available here as well.

I thought moving them to another package would be more complex since there probably will only be one dependent in the end.

Reply:
I changed the names to hopefully be a bit more descriptive and also added small documentation comments. Is this better now?

Reply:
yes :)